src/TypeScript/Sarif.SDK/package-lock.json

```diff
@@ -1,8 +1,1252 @@
 {
   "name": "sarif.sdk",
   "version": "2.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
+    "": {
+      "name": "sarif.sdk",
+      "version": "2.1.0",
+      "license": "MIT",
+      "dependencies": {
+        "@types/sarif": "^2.1.1"
+      },
+      "devDependencies": {
+        "@types/chai": "^4.1.3",
+        "@types/mocha": "^5.2.0",
+        "chai": "^4.1.2",
+        "mocha": "^9.2.2",
+        "ts-node": "^6.0.0",
+        "typescript": "^2.8.3"
+      }
+    },
+    "node_modules/@types/chai": {
+      "version": "4.1.3",
+      "resolved": "https://registry.npmjs.org/@types/chai/-/chai-4.1.3.tgz",
+      "integrity": "sha512-f5dXGzOJycyzSMdaXVhiBhauL4dYydXwVpavfQ1mVCaGjR56a9QfklXObUxlIY9bGTmCPHEEZ04I16BZ/8w5ww==",
+      "dev": true
+    },
+    "node_modules/@types/mocha": {
+      "version": "5.2.0",
+      "resolved": "https://registry.npmjs.org/@types/mocha/-/mocha-5.2.0.tgz",
+      "integrity": "sha512-YeDiSEzznwZwwp766SJ6QlrTyBYUGPSIwmREHVTmktUYiT/WADdWtpt9iH0KuUSf8lZLdI4lP0X6PBzPo5//JQ==",
+      "dev": true
+    },
+    "node_modules/@types/sarif": {
```

Comment:
>sarif [](http://example.com/codeflow?start=25&length=5)

first, this seems like an awful lot of churn in this file, is it expected? Also, why would we have a reference to the NPM SARIF types package? isn't that exactly what we're building in this code base?

Reply:
This file is generated by npm.

Reply:
@michaelcfanning Yes, this churn is par for the course. `package-lock.json` is practically considered opaque. I can dig deeper if you want to know why there's a self-reference.

Reply:
You folks are the experts, I am mostly here to learn and to try to nudge this one forward. If we think it's safe, let's add a release note and close on it.

Ed's asked a good question around what kind of test story we have to ensure we don't regress due to update. ?

Reply:
At a glance, the `TypeScript` folder/project does not produce the TypeScript typings nor any other meaningful output. I'm not sure why this is here. I presume this was the beginning of an effort to migrate the TypeScript types generator from the `sarif-vscode-extension` codebase to here.

We should probably delete this folder. The typings generator should migrate to the `sarif-js-sdk` now that it exists.

Reply:
@jeffersonking , with what you said in mind, is it safe to delete then?

Reply:
@eddynaka Yes, safe.

On closer inspection, It looks to be unfinished work from 33f83d68790966050c4b46f9da50ed1050087f23 and 6c9dc2347a305dbb1ef7ef664b58d3a2d0765ee8. @michaelcfanning would know this history, but I believe Everett's attempt was superseded by Rusty's.